docs/internal-documentation/maintenance/ReleaseProcess.md

```diff
@@ -71,38 +85,44 @@ CPAN].
 
 [ZNMSTR account at CPAN]: http://search.cpan.org/~znmstr/
 
-## 7. Verify that the module builds and all tests pass
+## 8. Verify that the module builds and all tests pass
 
 Verify that the module builds and all tests pass with the latest point release
 for every supported major Perl version and for every supported system locale.
-This can be done quite easily with something like this command for each locale:
+This can be done quite easily with something like this command for each locale
+and the list of supported Perl versions:
 
     LC_ALL=en.UTF-8 LC_MESSAGES=en.UTF-8 LC_NUMERIC=en.UTF-8 LANG=en.UTF-8 perlbrew exec --with 5.14.4,5.16.3,5.18.4,5.20.1 '( git clean -dfx && perl Makefile.PL && make ) >& /dev/null && prove -bQ'
```

Comment:
Version 5.22 and 5.24 should also be supported.

Should the list have the list patch version of Perl? If so:

5.14.4,5.16.3,5.18.4,5.20.3,5.22.4,5.24.3

(http://www.cpan.org/src/README.html)